### PR TITLE
Moving minimap to the right in POC

### PR DESCRIPTION
**Pull request by jpa399**
_Pull requests into Flux require the following. Submitter should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable (**N/A**) and :white_check_mark:. Reviewers should complete every item on the bulleted list when reviewing._

Moved the minimap to the right side of the targeted data panel in the poc version.
## Submitter:

**Running the Application**

- [x] Manually tested in Chrome (primary browser for testing)
- [ ] Manually tested in IE/Safari (verify app loads and basic feature testing)
- [x] Manually tested in Chrome in iPad mode (at minimum, viewing patient record)

**Task**

- [x] Branch implements task as expected
- [ ] Key application features are updated as needed for task and tested (see: [Key Application Features](https://github.com/FluxNotes/flux/wiki/Key-Application-Features))
- [x] Task specific definition of done (as described in the Jira task) is met, if applicable.

**Documentation**

- [x] This pull request describes why these changes were made
- [x] Recognizes any potential shortcomings/bugs in the description 
- [ ] Pre release script is updated on the [Pre Release Testing Script wiki page](https://github.com/FluxNotes/flux/wiki/Pre-Release-Testing-Script) (add/remove single features to test, do not run through full script)
- [ ] Update the list of key features with new features on the [Key Application Features wiki page](https://github.com/FluxNotes/flux/wiki/Key-Application-Features)
- [ ] Document any new APIs/extension points
- [ ] Additional technical components and libraries are documented and added to [wiki](https://github.com/FluxNotes/flux/wiki/About-Flux-Notes) as needed

**Code Quality**

- [x] Code style guide is followed (see: [Code Style Guide](https://github.com/FluxNotes/flux/wiki/JavaScript-and-HTML-Code-Style-Guide))
- [x] Code is commented

**Tests**

- [x] Existing tests passed
- [ ] Added backend tests for new functionality added

## Reviewers:

- Check that code is maintainable and reusable. Code reuses existing code and infrastructure where appropriate.
- Check the PR and code accomplishes the task's purpose.
- Check that new tests appropriately test the new code, including edge cases. Other existing tests pass.
- Verify Pull Request checklist is correctly completed by submitter.
- Try to break the code

### CODE REVIEW

**Comment by dreeves4321** [comment]
Looks great. I think we can scootch it to the right a bit more. Don't need more than 1 or 2 pixels at right of the grey overlay.
Seeing it on the right hand side made me think we do not need a vertical scrollbar. Does anybody have a reason not to remove it?

**Comment by dreeves4321** [comment]
I also see a marked jumpiness when dragging the minimap overlay, that's not there when scrolling in the TDP.  Does anybody else see that and/or know if that's fixable?

**Comment by jpa399** [comment]
Thanks!  I'll take a look at [DR1].  As for [DR2], I don't think I'm seeing the lag that you're seeing. Can someone else try on their computer? @mgramigna

**Comment by jpa399** [comment]
[DR1] resolved!  I left 1px to the right of the grey overlay.

**Comment by dreeves4321** [comment]
Just to clarify, I see that jumpiness on all branches and endpoints. Just noticed it now.

File: src/summary/flux.code-workspace [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (jafeltra) [inline src/summary/flux.code-workspace]: I think you can remove this file.

Comment (jpa399) [reply src/summary/flux.code-workspace]: all done!

File: src/lib/react-minimap/react-minimap.js [hunk 2]
```diff
@@ -114,11 +116,13 @@ export class Minimap extends React.Component {
 
               const wM = width * ratioX;
               let hM = Math.round(height * ratioY);
-              const xM = (left + scrollLeft - sourceRect.left) * ratioX;
+              const xM = this.props.isTablet? (sourceRect.width + 92 + 15) : ((left + scrollLeft - sourceRect.left) * ratioX); //92 is the width of the left navigation bar and 15 is buffer space
+              this.left = xM;
+              //const xM = this.props.isTablet? 700: (left + scrollLeft - sourceRect.left) * ratioX;
```
[the remark this answers is not in the document]
Comment (jpa399) [reply src/lib/react-minimap/react-minimap.js]: fixed!

**Comment by danlee1025** [comment]
Looks good, just needs a rebase.

**Comment by jpa399** [comment]
rebased!

**Pull request merged**